Shared/Phrases.cs

```diff
@@ -1,41 +1,240 @@
-﻿using Microsoft.Bot.Builder;
+﻿using System.Collections.Generic;

```

Comment:
Hey @sainianu08 is this PR in addition to the other one that is open? Also, it looks like the formatting on this file got a bit jumbled